### DISCUSSION TITLE
Is the idea of distributed table originated from ClickHouse?

### DISCUSSION THREAD

**Discussion by JackyWoo** [ClickHouse/ClickHouse#56159] [Q&A] [answered]
Just for curiosity. For the idea is much more different from distributed MPP execution architecture and it works well. If it is, may I known who raise it?

**Comment by alexey-milovidov** [comment, marked as answer]
Hi. The idea is mine; it was added here: https://github.com/ClickHouse/ClickHouse/commit/77c46cdcfd0bc3c3c714050df0c90e5d49ade6d6#diff-6f58cd6b3e578de39cb1de289bc6e06ae3c2b1b7751fcbf7f9f802fdad7fc762

**Reply by JackyWoo** [reply]
A really creative work.
